modules/conversion-center/classes/render/core-render.php

```diff
@@ -0,0 +1,217 @@
+<?php
+
+namespace Elementor\Modules\ConversionCenter\Classes\Render;
+
+use Elementor\Utils;
+
+/**
+ * Class Core_Render.
+ *
+ * This class handles the rendering of the Link In Bio widget for the core version.
+ *
+ * @since 3.23.0
+ */
+class Core_Render extends Render_Base {
```

Comment:
I am not sure that I understand yet how this render strategy will help with the re-usability? Because you pass a specific widget in here. But it may become clearer as we progress. :)

Reply:
@louiswol94 so the idea ( not set in stone of course, suggestions are always welcome :) ) is that when we implement variations, the Link_In_Bio class will become an abstract class and all widgets will subclass it and the Type Hinting will be for the abstract class. To be honest, this type hint could probably change to be `Widget_Base` as we don't use any method of our class. I would leave it like this for now and then this will be revisited based on how Variations are actually implemented. Let me know what you think